src/main/java/milk/manager/iot/Manager.java

```diff
@@ -0,0 +1,46 @@
+package milk.manager.iot;
+
+import milk.products.iot.MilkProduct;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class Manager {
+public List<MilkProduct> sortByPrice(final List<MilkProduct> milkbox,
+                                     final boolean descendingOrder) {
+    if (!descendingOrder) {
+        System.out.println("----------in increasing order"
+                + " by price----------\n");
+        return  milkbox.stream().sorted(Comparator
+                        .comparing(MilkProduct::getPriceInUahPerUnit))
+                        .collect(Collectors.toList());
+    }
+    System.out.println("----------in decreasing order by price----------\n");
+    return  milkbox.stream()
+                .sorted(Comparator.comparing(MilkProduct::getPriceInUahPerUnit)
+                    .reversed()).collect(Collectors.toList());
+}
+
+public List<MilkProduct> sortByOrigin(final List<MilkProduct> milkbox,
+                                      final boolean descendingOrder) {
+    if (!descendingOrder) {
+        System.out.println("----------in"
+                + " increasing order by origin----------\n");
+        return  milkbox.stream().sorted(Comparator
+                .comparing(MilkProduct::getOrigin))
+                .collect(Collectors.toList());
+    }
+    System.out.println("----------in decreasing order by origin----------\n");
+    return  milkbox.stream().sorted(Comparator
+            .comparing(MilkProduct::getOrigin)
+            .reversed()).collect(Collectors.toList());
+
+}
+public List<MilkProduct> searchByHomeMadeBoolean(final List<MilkProduct> milk) {
+    System.out.println("----------Here are"
+            + " products that are made at home----------\n");
+    return milk.stream().
+            filter(MilkProduct::getHomeMade).collect(Collectors.toList());
+}
+}
```

Comment:
Pay attention to **tabulation** and remove redundant **blank lines**.
Add the tab to the class methods (_lines 10-45_). 

Here's what Oracle documentation says:
**6.4 Class and Interface Declarations**
When coding Java classes and interfaces, the following formatting rules should be followed:

``` java
class Sample extends Object {
    int ivar1;
    int ivar2;

    Sample(int i, int j) {
        ivar1 = i;
        ivar2 = j;
    }

    int emptyMethod() {}

    ...
}
class Sample extends Object {
    int ivar1;
    int ivar2;

    Sample(int i, int j) {
        ivar1 = i;
        ivar2 = j;
    }

    int emptyMethod() {}

    ...
}
```
Methods are separated by a blank line